lib/systems/parse.nix

```diff
@@ -402,7 +402,7 @@ rec {
       else if hasPrefix "genode" (elemAt l 2)
         then { cpu = elemAt l 0; vendor = elemAt l 1; kernel = elemAt l 2; }
       else throw "Target specification with 3 components is ambiguous";
-    "4" =    { cpu = elemAt l 0; vendor = elemAt l 1; kernel = elemAt l 2; abi = elemAt l 3; };
+    "4" =    { cpu = elemAt l 0; vendor = if elemAt l 1 == "oe" then "unknown" else elemAt l 1; kernel = elemAt l 2; abi = elemAt l 3; };
```

Comment:
Yes, it's a hack. I'd like to know how this can be done differently.

Reply:
If we do want to use the binary toolchain, we have to deal with the fact that its target specifications are different from Nixpkgs's.  I couldn't find any way to do this without either hackery in parse.nix or overriding the parsed representation of the config itself, which is the main reason why I didn't try to upstream the configuration in the first place.